### ISSUE TITLE
Request - ANGLE implementation for UWP RA cores (Xbox) support

### ISSUE THREAD

**Issue by Player1LoginXBOX** [dhewm/dhewm3#468]
Correct me if I'm wrong but the OpenGL requirement for this core and boom 3 xp core are low enough at 2.0 and undemanding enough that they should work smooth enough with ANGLE implemented into the core and would allow these 2 RetroArch cores to be enjoyed on UWP (Xbox) developer mode.

**Comment by DanielGibson** [comment]
I don't know what Boom 3 xp is, and this is not a retroarch core, but a normal standalone source port of Doom3.

ANGLE only supports OpenGL ES, while dhewm3 uses "normal" Desktop OpenGL, more specifically (legacy) OpenGL 1.4 with the [ARB shaders](https://en.wikipedia.org/wiki/ARB_assembly_language) extension. That's not supported by ANGLE and porting to it wouldn't be easy.
There is a fork of dhewm3 that uses OpenGL ES2 to run in browsers (with WebGL), but it doesn't support running natively on Windows: https://github.com/gabrielcuvillier/d3wasm
Furthermore its renderer is a bit incomplete, not all shader effects have been ported completely, for example the heat haze postprocessing effect is missing.

**Comment by Player1LoginXBOX** [comment]
Thanks for eli5 @DanielGibson
